logger/module.go

```diff
@@ -9,18 +10,18 @@ import (
 )
 
 type loggerModule struct {
+	zerolog.Logger
```

Comment:
Can you replace this with `logger zerolog.Logger`?

1. I don't believe `Logger` needs to be publically exposed, does it?
2. It makes it easier to track/understand the names of vars if we don't use implicit naming.

Reply:
The idea was to make `loggerModule` be an extension of  `zerolog.Logger` instead of having a `zerolog.Logger` property. This way the we sould be able to use `loggerModule` directly to make the zerolog calls like `.Debug()`, `.Info()`, etc.

Reply:
Got it. You're making use of what's called "promoted fields"